src/doc/unstable-book/src/compiler-flags/env.md

```diff
@@ -5,7 +5,10 @@ The tracking issue for this feature is: [#118372](https://github.com/rust-lang/r
 ------------------------
 
 This option flag allows to specify environment variables value at compile time to be
-used by `env!` and `option_env!` macros.
+used by `env!` and `option_env!` macros. It also impacts `tracked_env::var` function
+from the `proc_macro` crate.
+
+This information will be stored in the dep-info files.
```

Comment:
Might be good to add a link to an explanation what dep-info files are for those who don't know. This one looks good enough to me: https://doc.rust-lang.org/cargo/guide/build-cache.html#dep-info-files

Reply:
Yes indeed!